<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevicePropertiesDialog</class>
 <widget class="QDialog" name="DevicePropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>446</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Device Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabProperties">
      <attribute name="title">
       <string>Properties</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="nameLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Device name:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="lineEdit_DeviceName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="profileLayout">
         <item row="1" column="0">
          <widget class="QComboBox" name="comboBox_Profile">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentText">
            <string>[Default]</string>
           </property>
           <item>
            <property name="text">
             <string>[Default]</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelPRofile">
           <property name="text">
            <string>Profile</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_Edit">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="lightLayout">
         <item row="0" column="1">
          <widget class="QCheckBox" name="checkBox_AutoLightColor">
           <property name="text">
            <string>Automatic light color</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox_UseProfileLight">
           <property name="text">
            <string>Use profile light</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="colorLayout">
         <item row="1" column="1">
          <widget class="QSlider" name="horizontalSlider_Green">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="horizontalSlider_Blue">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelRed">
           <property name="text">
            <string>Red</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelGreen">
           <property name="text">
            <string>Green</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelBlue">
           <property name="text">
            <string>Blue</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="pushButton_CustomColor">
           <property name="text">
            <string>Custom</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="horizontalSlider_Red">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_UseProfileIdle">
         <property name="text">
          <string>Use profile idle settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_IdleOptions">
         <property name="title">
          <string>Idle options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBox_IdleDisconnect">
            <property name="text">
             <string>Disconnect on idle</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridIdle">
            <item row="0" column="0">
             <widget class="QDoubleSpinBox" name="doubleSpinBox">
              <property name="frame">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_IdleUnit">
              <property name="currentText">
               <string>Minutes</string>
              </property>
              <property name="currentIndex">
               <number>1</number>
              </property>
              <item>
               <property name="text">
                <string>Seconds</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Minutes</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Hours</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabReadout">
      <attribute name="title">
       <string>Readout</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupAnalogSticks">
         <property name="title">
          <string>Analog Sticks</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>LX</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelLX">
            <property name="text">
             <string>Value</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>LY</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelLY">
            <property name="text">
             <string>Value</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>RX</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="labelRX">
            <property name="text">
             <string>Value</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>RY</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="labelRY">
            <property name="text">
             <string>Value</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupTriggers">
         <property name="title">
          <string>Triggers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupTouchA">
         <property name="title">
          <string>Touch A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupTouchB">
         <property name="title">
          <string>Touch B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupGyroscope">
         <property name="title">
          <string>Gyroscope</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupAccelerometer">
         <property name="title">
          <string>Accelerometer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupLatency">
         <property name="title">
          <string>Latency</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
